app/Http/Controllers/EntityCreatorController.php

```diff
@@ -104,15 +116,24 @@ public function store(Request $request, $type)
                 continue;
             }
             $values['name'] = $name;
-
-            $this->validateEntity($values, $validator->rules());
-
-            /** @var MiscModel $model */
-            $model = new $class;
-            $new = $model->create($values);
-            $new->crudSaved();
-            $new->entity->crudSaved();
-
+            if ($type != 'posts') {
+                $this->validateEntity($values, $validator->rules());
+
+                /** @var MiscModel $model */
+                $model = new $class;
+                $new = $model->create($values);
+                $new->crudSaved();
+                $new->entity->crudSaved();
+            } else {
+                if ($values['position'] == 0) {
```

Comment:
Worth adding a comment before the if to explain the difference between the if and else, so it's faster to parse :) I wasn't sure if the first if is for position first or position last without having to look deeper in the condition